code/game/objects/effects/poster.dm

```diff
@@ -20,6 +20,10 @@
 	var/poster_type
 	var/obj/structure/sign/poster/poster_structure
 
+/obj/item/poster/examine(mob/user)
+	. = ..()
+	. += span_notice("You can trap it with a glass shard")
```

Comment:
```suggestion
	. += span_notice("You can booby-trap the poster by using a glass shard on it before you put it up.")
```

Something like this helps explain how exactly to go about booby-trapping the poster.